### DISCUSSION TITLE
Gosund SP111 - Unable to connect

### DISCUSSION THREAD

**Discussion by przemoc86** [make-all/tuya-local#611] [Q&A] [answered]
Hello, I'm not able to add Gosund SP111 socket. It was working on localTuya, but I would like to switch all my Tuya devices to this addon. Uuid and key are for sure correct - working on previously mentioned localTuya and I double checked it on IoT Tuya platform.
I've tried with auto IP setup and with all Protocol versions - result always the same.

![2023-04-14 01_16_07-Settings – Home Assistant – Brave](https://user-images.githubusercontent.com/62384501/231902892-c36fa314-2390-498a-92ad-b2186fe7c6ac.png)

**Comment by make-all** [comment, marked as answer]
See #522

**Reply by przemoc86** [reply]
Thanks - I'll try those steps today and write an update

**Comment by przemoc86** [comment]
Hello, it's working :)
As wrote in #522 , I've paired both sockets once again, removed them from localtuya and added them to tuya-local with tuya smart app closed. Thanks for quick help
